### ISSUE TITLE
What is the article for the "Multi Camera Multi Target Python* Demo" ?

### ISSUE THREAD

**Issue by hoangkhoiLE** [openvinotoolkit/open_model_zoo#1497]
Hello,

I want to learn more about the code and the way it work. Please share to me some of document relating with "https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/python_demos/multi_camera_multi_target_tracking".

Best regard

**Comment by vladimir-dudnik** [comment]
which document do you mean? What we currently have is the code and brief demo description on github. This should be available, I think.

**Comment by hoangkhoiLE** [comment]
I mean document to describe how and which algorithm of tracking that you used. I want to modify some of parameter in the code but there is no document to follow. 

Best regard.

**Comment by vladimir-dudnik** [comment]
@hoangkhoiLE I do not think we have any additional documents related to this sample. @evgeny-izutov @snosov1 do you mind to describe tracking algorithm in more details?

**Comment by hoangkhoiLE** [comment]
I always wait for your advices.

Best regard

**Comment by evgeny-izutov** [comment]
@DmitriySidnev, @sovrasov

**Comment by DmitriySidnev** [comment]
@hoangkhoiLE, hi!
The main algorithm of the `multi_camera_multi_target_tracking` is based on a re-identification model which extracts a re-id feature from every bounding box. Also there is used additional heuristics such as IoU between bounding boxes, velocity constraint e.t.c. Finally we did not write an article, we used already known methods. We were focused on re-identification models in this algorithm.
